notebooks/Understanding_WFSim_P1.ipynb

```diff
@@ -27,7 +27,7 @@
    "outputs": [],
    "source": [
     "config = straxen.get_resource('https://raw.githubusercontent.com/XENONnT/'\n",
-    "                 'strax_auxiliary_files/master/fax_files/fax_config_nt.json', fmt='json')\n",
+    "                 'strax_auxiliary_files/master/fax_files/fax_config_nt_design.json', fmt='json')\n",
```

Comment:
I got the link from there

Reply:
to be fair, the line from before also doesn't work

Reply:
Ah, oops 😄 I thought you were just looking at the script. Indeed in the notebooks I just changed `nt` by `nt_design` from the command line, so completely overlooked that this link was too old to work (`fax_files` instead of `sim_files`, public repo...).

I close this PR and will fix the tutorials once #111 is in place, since I think other settings need to be updated besids this one.

PS: Shame on me, when you read the first three words I wrote in this PR.

Reply:
Sounds good, I think that would make sense, thanks Diego!